just/bluefin-system.just

```diff
@@ -280,3 +280,49 @@ update-ng:
     sudo bootc upgrade
     flatpak update -y
     brew upgrade
+
+# Toggle tailscale
+toggle-tailscale:
+    #!/bin/bash
+    source /usr/lib/ujust/ujust.sh
+
+    TAILSCALED_STATUS="$(systemctl is-enabled tailscaled)"
+
+    if [ "$TAILSCALED_STATUS" == "enabled" ]; then
+        TAILSCALED="Installed"
+    elif [ "$TAILSCALED_STATUS" == "disabled" ]; then
+        TAILSCALED="Installed"
+    else 
+        echo "${b}${red}Unable to enable or disable Tailscale.${n}"
+        echo "tailscaled service status: $TAILSCALED_STATUS"
```

Comment:
This should be saying cannot find tailscaled.

Reply:
Per the documentation of [is-enabled](https://www.man7.org/linux/man-pages/man1/systemctl.1.html), there are other potential output statuses that could be returned which I haven't tested for, some of which would be returned in scenarios where the unit file is present. So my concern is that stating "Cannot find tailscaled" might not necessarily be accurate, because sometimes the command could fail despite recognizing the presence of of tailscaled. My approach was to instead echo the service status returned by systemctl, informing the user of the status and allowing them to address the problem through methods such as unmasking the unit file.

Do you think I should rewrite the check, continue with changing the message to "Cannot find the tailscaled service.", or leave it as-is?

Reply:
Leave as is then.